src/main/java/org/mqttbee/util/UnsignedDataTypes.java

```diff
@@ -26,6 +26,10 @@
     public static final long UNSIGNED_INT_MAX_VALUE = 0xFFFF_FFFFL;
 
     public static boolean isUnsignedShort(final int value) {
+        return isUnsignedShort((long) value);
```

Comment:
The cast will happen automatically if calling isUnsignedShort(long value) with an int as argument.
It would make sense if you duplicated the code `return (value >= 0) && (value <= UNSIGNED_SHORT_MAX_VALUE);` to not have to cast and operate on long.
Otherwise remove the method.

Reply:
OK, I'll remove the int version of the function.